pkg/deployment/depresource/new_stateless.go

```diff
@@ -30,7 +30,7 @@ import (
 type NewStateless struct {
 	*api.API
 
-	// Required: DeploymentID.
+	// Optional DeploymentID.
```

Comment:
How come?

Reply:
Because a new deployment doesn't have an ID yet 😄 

From what I wrote on the description ->
> I have added params.DeploymentID != "" && len(params.DeploymentID) != 32 to the NewStateless validation as a temporary measure until the above has been implemented.

This will be cleaned up when the `deployment update` flags issue is implemented.